build/bin/release

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [ "$#" -lt "1" ]; then
+    echo "must specify chart directory"
+    exit 1
+fi
+
+set -x
+set -e
+
+chart=$1
+
+rm -r build/release
+
+owner=${GITHUB_USER:-onosproject}
+repo=${GITHUB_REPO:-onos-helm-charts}
+remote=${GIT_REMOTE:-origin}
+token=${GITHUB_TOKEN}
+branch=$(git branch --show-current)
+
+# Package the Helm chart
+helm package $chart --destination build/release
+
+# Upload the Helm chart release
+cr upload \
+    --git-base-url https://api.github.com/ \
```

Comment:
I get `Error: unknown flag: --git-base-url` when running this with `cr` version *v0.2.3*

Reply:
This seems to be a new feature and this is the default value anyways, so I'm removing it